src/EnergyPlus/api/runtime.h

```diff
@@ -56,6 +56,7 @@
 
 // if we are building this with C++ in mind, we can make use of these std::function based callback registration functions
 #include <functional>
+#include <string>
```

Comment:
missing import. Needed or when using the exported lib on ubuntu complains that std::string isn't defined (so does clang-tidy warnings for that matter)

Reply:
Got it.